base/utf8ienc.dtx

```diff
@@ -1368,6 +1368,18 @@
 %<all,t1,ly1>\DeclareUnicodeCharacter{017E}{\v z}
 %<all,ts1,ly1>\DeclareUnicodeCharacter{0192}{\textflorin}
 %    \end{macrocode}
+% \changes{v1.2n}{2021/12/10}{add 01C4-01CC}
+%    \begin{macrocode}
+%<all,t1>\DeclareUnicodeCharacter{01C4}{D\v Z}
+%<all,t1>\DeclareUnicodeCharacter{01C5}{D\v z}
+%<all,t1>\DeclareUnicodeCharacter{01C6}{d\v z}
```

Comment:
I kow that are are still people using LY1 and all such definitions would work for them too wouldn't they? so perhaps we should always set all,T1,LY1 if that is possible. (might also apply to other chars we added recently).

Reply:
@FrankMittelbach  and OT1 but actually it doesn't make any difference these days as we pre-load OT1 and T1 into the format so any UTF-8 declaration in T1 is available whatever encodings you use.

Reply:
true, so forget my remark (which you probably already did :-))